### PR TITLE
added supra to oracle documentation

### PR DESCRIPTION
**Pull request by ksdumont**
Supra launched oracle services to Polygon mainnet. I updated your oracle documentation to add Supra to the list.

### CODE REVIEW

File: docs/tools/oracles/supra.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+!!! info "Content disclaimer"
+
+    Please view the third-party content disclaimer [here](https://github.com/0xPolygon/wiki/blob/master/CONTENT_DISCLAIMER.md).
+
+**Supra** is a novel, high-throughput Oracle & IntraLayer: A vertically integrated toolkit of cross-chain solutions (data oracles, asset bridges, automation network, and more) that interlink all blockchains, public (L1s and L2s) or private (enterprises).
```
Comment (kmurphypolygon) [inline docs/tools/oracles/supra.md]:
```suggestion
**Supra** is a novel, high-throughput oracle & intralayer: A vertically integrated toolkit of cross-chain solutions (data oracles, asset bridges, automation network, and more) that interlink all blockchains, public (L1s and L2s) or private (enterprises).
```

File: docs/tools/oracles/supra.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+[Supra](https://supraoracles.com/) provides decentralized oracle price feeds that can be used for on-chain and off-chain use-cases such as spot and perpetual DEXes, lending protocols, and payments protocols. Supra’s oracle chain and consensus algorithm makes it the fastest-to-finality oracle provider, with layer-1 security guarantees. The pull oracle has a sub-second response time. Aside from speed and security, Supra’s rotating node architecture gathers data from 40+ data sources and applies a robust calculation methodology to get the most accurate value. The node provenance on the data dashboard also provides a fully transparent historical audit trail. Supra’s Distributed Oracle Agreement (DORA) paper was accepted into ICDCS 2023, oldest distributed systems conference.
```
Comment (kmurphypolygon) [inline docs/tools/oracles/supra.md]:
```suggestion
[Supra](https://supraoracles.com/) provides decentralized oracle price feeds that can be used for on-chain and off-chain use-cases such as spot and perpetual DEXes, lending protocols, and payments protocols. Supra’s oracle chain and consensus algorithm makes it the fastest-to-finality oracle provider, with layer-1 security guarantees. The pull oracle has a sub-second response time. Aside from speed and security, Supra’s rotating node architecture gathers data from 40+ data sources and applies a robust calculation methodology to get the most accurate value. The node provenance on the data dashboard also provides a fully transparent historical audit trail. Supra’s Distributed Oracle Agreement (DORA) paper was accepted into ICDCS 2023, the oldest distributed systems conference.
```

File: docs/tools/oracles/supra.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+Check out our developer docs [here](https://supraoracles.com/docs/overview/).
```
Comment (kmurphypolygon) [inline docs/tools/oracles/supra.md]:
```suggestion
Check out our [developer docs](https://supraoracles.com/docs/overview/).
```

**Review by kmurphypolygon — CHANGES_REQUESTED** [review]
A few tiny changes please.

**Comment by ksdumont** [comment]
I committed all of your changes, kindly let me know when it is merged.

**Review by kmurphypolygon — APPROVED** [review]

**Comment by kmurphypolygon** [comment]
Merged and thank you @ksdumont

**Pull request merged**